homeassistant/components/tesla_fleet/sensor.py

```diff
@@ -378,6 +378,12 @@ class TeslaFleetTimeEntityDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.POWER,
         entity_registry_enabled_default=False,
     ),
+    SensorEntityDescription(
+        key="island_status",
+    ),
+    SensorEntityDescription(
+        key="storm_mode_active",
+    ),
```

Comment:
```suggestion
    SensorEntityDescription(
        key="storm_mode_active",
        options=["island_status_unknown","on_grid","off_grid","off_grid_unintentional","off_grid_intentional"],
        device_class=SensorDeviceClass.ENUM,
    ),
```

Reply:
Just a question, but shouldn’t that be for island_status?

Reply:
Yep, in my rush I did it on the wrong line. Good catch.

Reply:
Fixed in [8ef08a5](https://github.com/home-assistant/core/pull/126438/commits/8ef08a5931e400dc877e3df7bc3eec4f2987d832)